tests/helper/tests/sgid_suid_files.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+def sgid_suid_files(client, id_type, whitelist_files):
+    """ Performing unit test for feature: Fedramp """
+    if id_type == 'sgid':
+        remote_files = _get_remote_files(client, "-2000")
+        _val_whitelist_files(remote_files, whitelist_files)
+    if id_type == 'suid':
+        remote_files = _get_remote_files(client, "-4000")
+        _val_whitelist_files(remote_files, whitelist_files)
+
+
+def _get_remote_files(client, perm):
+    """ Get remote files of given attribute """
+    files = []
+    cmd_find = f"find / -type f -perm {perm} -exec "
+    cmd_stat = "stat -c '%n,%u,%g' {} \; 2> /dev/null"
+    cmd = cmd_find + cmd_stat
+    (exit_code, output, error) = client.execute_command(
+        cmd, quiet=True)
+
+    for line in output.split('\n'):
+        if line != '':
+            files.append(line)
+    return files
+
+
+def _val_whitelist_files(remote_files, whitelist_files):
+    """ Validates that remotly found files are in whitelist """
+    found_files = []
+    for file in remote_files:
+        if file not in whitelist_files:
+            found_files.append(file)
+            logger.warn(f"Found file: {file}")
```

Comment:
this is working without importing logging and initializing the logger?

Reply:
Ah, thanks. Removed the logging line. This was for testing purpose.